plugins/org.locationtech.udig.project/src/org/locationtech/udig/project/internal/render/impl/TiledRendererCreatorImpl.java

```diff
@@ -321,6 +321,18 @@ public SelectionLayer findSelectionLayer( ILayer targetLayer ) {
         throw new UnsupportedOperationException("Renderers don't have information in this system."); //$NON-NLS-1$
     }
 
+    /**
+     * @throws UnsupportedOperationException
```

Comment:
Where is a UnsupportedOperationException thrown?

Reply:
You are right I will remove it